From 7a8d50c2813cb9ece023816e1b4efe1d8556a4f8 Mon Sep 17 00:00:00 2001
From: Markus Armbruster <armbru@redhat.com>
Date: Mon, 11 Aug 2014 07:54:44 +0200
Subject: [PATCH 10/16] rhel: nec-usb-xhci.superspeed-ports-first=off for machine types <= 7.0

Message-id: <1407743689-13553-11-git-send-email-armbru@redhat.com>
Patchwork-id: 60509
O-Subject: [PATCH RHEV-7.1 qemu-kvm-rhev 10/15] rhel: nec-usb-xhci.superspeed-ports-first=off for machine types <= 7.0
Bugzilla: 1118665
RH-Acked-by: Eduardo Habkost <ehabkost@redhat.com>
RH-Acked-by: Miroslav Rezanina <mrezanin@redhat.com>
RH-Acked-by: Dr. David Alan Gilbert (git) <dgilbert@redhat.com>

Update RHEL compatibility properties for upstream commit 7bafd88.

Signed-off-by: Markus Armbruster <armbru@redhat.com>
---
 include/hw/i386/pc.h | 4 ++++
 1 file changed, 4 insertions(+)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 include/hw/i386/pc.h |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/include/hw/i386/pc.h b/include/hw/i386/pc.h
index 8be4cf4..226390f 100644
--- a/include/hw/i386/pc.h
+++ b/include/hw/i386/pc.h
@@ -537,6 +537,10 @@ bool e820_get_entry(int, uint32_t, uint64_t *, uint64_t *);
             .property = "version",\
             .value    = stringify(0x11),\
         },{\
+            .driver   = "nec-usb-xhci",\
+            .property = "superspeed-ports-first",\
+            .value    = "off",\
+        },{\
             .driver   = "PIIX4_PM",\
             .property = "acpi-pci-hotplug-with-bridge-support",\
             .value    = "off",\
-- 
1.7.1

